src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/XamarinAndroidCommonProject.cs

```diff
@@ -54,8 +54,12 @@ protected XamarinAndroidCommonProject (string debugConfigurationName = "Debug",
 			AndroidResources.Add (new AndroidItem.AndroidResource ("Resources\\drawable-xxhdpi\\Icon.png") { BinaryContent = () => icon_binary_xxhdpi });
 			AndroidResources.Add (new AndroidItem.AndroidResource ("Resources\\drawable-xxxhdpi\\Icon.png") { BinaryContent = () => icon_binary_xxxhdpi });
 			//AndroidResources.Add (new AndroidItem.AndroidResource ("Resources\\drawable-nodpi\\Icon.png") { BinaryContent = () => icon_binary });
+			if (Builder.UseDotNet) {
+				// set our default
+				SetProperty ("AndroidUseDesignerAssembly", "True");
+			}
```

Comment:
If this is on by default in our `.targets`, do we need this?

Reply:
This is required because in `BaseTest.cs` [1] we check to see if this is enabled or not. Our "in memory" representation of the project does not load the defaults from the .targets, so if we query this property via `project.GetProperty ("AndroidUseDesignerAssembly")` we could get and empty back. Which does not reflect what is actually set.

[1] https://github.com/xamarin/xamarin-android/pull/6427/files/20f3cf4a44d5860bf9accfe5b9a5e37d573142f6#diff-fc6b8264d09940f48cf877e99ca193aed8feb1f9bddde5a46e752cc12b638d3aR573